src/components/organisms/Register/index.tsx

```diff
@@ -0,0 +1,184 @@
+'use client';
+
+import RegisterChild from '@/components/molecules/RegisterChild';
+import styles from './Register.css';
+import Button from '@/components/atoms/Button';
+import Typography from '@/components/atoms/Typography';
+import { useState } from 'react';
+import Loader from '@/components/atoms/Loader';
+import { Register as RegisterProps } from './types';
+
+export const links = () => {
+  return [{ rel: 'stylesheet', href: styles }];
+};
+
+type Props = RegisterProps & React.HTMLAttributes<HTMLElement>;
+
+const Register = ({ bankAccount, leaderPrice, childPrice }: Props) => {
+  const [responseMessage, setResponse] = useState('');
+
+  const handleSubmit = async (event: any) => {
+    // Stop the form from submitting and refreshing the page.
+    event.preventDefault();
+    setResponse('loading');
+
+    const basedata = {
+      street: event.target.elements.street.value,
+      houseNumber: event.target.elements.number.value,
+      busNumber: event.target.elements.bus.value,
+      arealCode: event.target.elements.Postcode.value,
+      city: event.target.elements.City.value,
+      email: event.target.elements.email.value,
+      phoneNumber: event.target.elements.tel.value,
+    };
+
+    // for each child do an api call
+
+    for (let i = 1; i < 10; i++) {
+      if (!event.target.elements['firstname' + i]) break;
+      const extraData = {
+        firstName: event.target.elements['firstname' + i].value,
+        lastName: event.target.elements['lastname' + i].value,
+        birthDate: event.target.elements['birthdate' + i].value,
+        akabe: event.target.elements['akabe' + i].checked,
+        sex: event.target.elements['Sex' + i].value,
+      };
+
+      const fullData = { ...basedata, ...extraData };
+      const JSONdata = JSON.stringify(fullData);
+      const options = {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSONdata,
+      };
+      const response = await fetch('/api/register_user', options);
+      const result = await response.json();
+      if (response.status !== 200) {
+        alert(
```

Comment:
Dit hoeft geen alert te zijn, de user heeft niks aan deze feedback. We maken hier nog een toast notification van met meaningful copy. Log dit voorlopig maar gewoon in de console.